SimG4Core/PrintGeomInfo/test/python/g4OverlapCheck2026DDD_cfg.py

```diff
@@ -2,7 +2,7 @@
 # Way to use this:
 #   cmsRun g4OverlapCheck2026_cfg.py geometry=D86 tol=0.1
```

Comment:
Perhaps for a privately used config this update of a comment line is not necessary:
```suggestion
#   cmsRun g4OverlapCheck2026DDD_cfg.py geometry=D86 tol=0.1
```

@bsunanda please let us know if you want to update it, or if you agree merging the PR as it is, instead

Reply:
@perrotta , I would guess DD4hep is our default for these checks, not DDD, but Sunanda may clarify.

Reply:
> @perrotta , I would guess DD4hep is our default for these checks, not DDD, but Sunanda may clarify.

@civanch tbat's not the point. The point is that in the instruction to run it the name of the file is wrong. This is a very minor issue for a script that has to be run privately, by experts I imagine. In any case, if @bsunanda agrees we can even fix it and merge without retriggering tests and signature, as it is just a comment line